### PR TITLE
Other pages

### PR DESCRIPTION
**Pull request by cwulfman**
Adddresses #13.

- Definition of Types has been incorporated into the types page;
- The entries in the comment board have been migrated to a new page, blog.html.

Yet to be converted is the sources page.

### CODE REVIEW

**Comment by cwulfman** [comment]
Sources page has been converted.

File: pages/subjects.md [hunk 1]
```diff
@@ -7,519 +7,603 @@ permalink: /library_types.html
...
+                these colleges have been considered as agricultural colleges.</dd>
+            <dt>Agricultural library belonging to a state government. </dt>
+            <dd>A few state boards of agriculture had libraries, beginning in the 1850s.</dd>
+            <dt>Agricultural social library.</dt>
+            <dd> A library belonging to a group that was organized to establish a library about
+                agriculture.</dd>
+            <dt>Agricultural society library. </dt>
+            <dd>A library owned by a society formed because its members were interested in
+                agriculture. Some of these were called <i>Farmers' clubs</i>.</dd>
+            <dt<>Antiquarian library. </dt>
```
Comment (escowles) [inline pages/subjects.md]: Looks like a typo in the `<dt>` tag

File: pages/blog.md [hunk 2]
```diff
@@ -0,0 +1,1465 @@
...
+                1824</div>
+
+        </div>
+        <div class="comment">
+            <div class="header">On 2010-08-27 21:05:02 <a href="mailto:stustiff@verizon.net">stuart
+                    stiffler</a> wrote:</div>
+            <div class="body"><b>Library Name: </b>Nelson Library Company<br/><b>Location:
+                </b>Nelson, Portage County, Ohio<br/><br/> In re founding date:1829----library was
+                established
+                "probaBLY<br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/><br/> IN
```
Comment (escowles) [inline pages/blog.md]: There are a dozen or so places with several `<br/>` tags in a row creating pretty big gaps in the page. Maybe trim everything down to at most 2 `<br/>`'s in a row?

**Comment by escowles** [comment]
@cwulfman This is looking good — I flagged two very small things but otherwise think we can ship this!

**Pull request merged**